sherpa-onnx/csrc/offline-tts.h

```diff
@@ -91,6 +99,8 @@ class OfflineTts {
   // If it supports only a single speaker, then it return 0 or 1.
   int32_t NumSpeakers() const;
 
+  std::unique_ptr<OfflineTtsCacheMechanism> cache_mechanism_; // not owned here
```

Comment:
```suggestion
```

Please don't add any new fields to OfflineTts.

As said before, we use `pimpl` here.

If you don't know what `pimpl` is, please google it first.

The only member `OfflineTts` has is `impl_`.

Everything else is put in`OfflineTtsImpl`.